package.json

```diff
@@ -277,6 +277,7 @@
   "devDependencies": {
     "@babel/core": "7.16.0",
     "@babel/plugin-proposal-export-namespace-from": "7.16.0",
+    "@brave/brave-ui": "0.40.1",
```

Comment:
should this actually be in deps instead of devdeps?

Reply:
I wonder if the old one was a devDependency because we were compiling it? Not sure though

Reply:
I suppose the only(?) difference between devDependencies and dependencies for brave-core is whether we check for a LICENSE file during build. So in that sense brave-ui should be in dependencies.  Changed